backend/models/door.go

```diff
@@ -2,7 +2,7 @@ package models
 
 type Door struct {
 	Id          uint   `json:"id"`
-	Name        string `json:"name"`
+	Name        string `json:"name" gorm:"unique"`
```

Comment:
why? people have the same name all the time, and we never query off name I think this is unnecessary and will just make it more of a pain for testing